### ISSUE TITLE
null in params is converted to an empty object

### ISSUE THREAD

**Issue by zabavnikov** [acro5piano/typed-graphqlify#187]
Code:
![image](https://user-images.githubusercontent.com/8766240/100566491-ca2a9a80-32f8-11eb-84ad-86aba903bc54.png)

Error: (I expect null)
![image](https://user-images.githubusercontent.com/8766240/100566660-32797c00-32f9-11eb-990f-42fe6c7b5dd8.png)

**Comment by zabavnikov** [comment]
Now I use this solution
![image](https://user-images.githubusercontent.com/8766240/100572663-a1aa9c80-3308-11eb-981e-d7594b6dcb19.png)

**Comment by acro5piano** [comment]
Sorry for the delay. 

# Solution 1: use GraphQL operation variables

The better solution.

```ts
const travelsQuery = gql('query($page: Int!, $top: Int!)', {
  [alias('travels', 'travels(page: $page, top: $top)']: {
    data: {
      id: types.number,
    }
  }
})
axios.post({
  query: travelsQuery.toString(),
  variables: { page: page || 1, top: top || null,  }
})
```


# Solution 2: alias

How about using `alias`?

```ts
const travelsQuery = gql({
  [alias('travels', `travels(page: ${query.page || 1}, top: ${query.top || 'null'})`]: {
    data: {
      id: types.number,
    }
  }
})
```

**Comment by acro5piano** [comment]
I'll try to render `null` if the param is null.

**Comment by acro5piano** [comment]
@zabavnikov I've fixed and published v3.0.1 which includes the fix. Please try it out! Thanks.
